app/views/challenge_signups/_signup_form.html.erb

```diff
@@ -22,11 +22,14 @@
 <% end %>
 
 <%= form_for([@collection, @challenge_signup], :url => (@challenge_signup.new_record? ? collection_signups_path(@collection) : collection_signup_path(@collection))) do |signup_form| %>
-  <% unless @current_user&.preference&.allow_gifts? %>
-    <p class="caution notice"><%= ts("Signing up for this challenge will allow users to give you gift works in response to your requests. However, you can always %{refuse_faq}.", refuse_faq: link_to("refuse a gift", archive_faq_path("your-account", anchor: "refusegift"))).html_safe %></p>
+  <% if !@current_user&.preference&.allow_gifts? && @collection.challenge_type == "GiftExchange" %>
+    <p class="caution notice"><%= ts("Signing up for this challenge will allow assigned users to gift works to you regardless of your preference settings. If you wish to receive additional gifts from users who are not assigned to you, please %{preferences} to allow gifts from anyone. You can always %{refuse_faq}.", preferences: link_to("update your preferences", user_preferences_path(@current_user)), refuse_faq: link_to("refuse a gift", archive_faq_path("your-account", anchor: "refusegift"))).html_safe %></p>
```

Comment:
We're not turning these into proper i18n keys yet, but let's follow our convention and have the placeholder variables use the suffix "_link".

Reply:
> We're not turning these into proper i18n keys yet, but let's follow our convention and have the placeholder variables use the suffix "_link".

Awesome thanks